Makefile

```diff
@@ -80,6 +80,9 @@ generate-swagger: build-swagger-fix
 	rm -rf dataplane/api/client dataplane/api/model
 	swagger generate client -f http://$(CB_IP):$(CB_PORT)/cb/api/swagger.json -c client -m model -t dataplane/api
 	make fix-swagger
+generate-swagger-dp: 
```

Comment:
Dockerized generator is missing

Reply:
I will create a defect for that. will good it as part of M3

Reply:
https://hortonworks.jira.com/browse/BUG-115256